operator/api/v1alpha1/kyma_types.go

```diff
@@ -224,11 +218,11 @@ type ModuleInfo struct {
 	// ModuleName is the unique identifier of the module.
 	ModuleName string `json:"moduleName"`
 
-	// Channel is the current deployed module channel
-	Channel Channel `json:"channel"`
-
-	// GroupVersionKind is the current deployed module gvk
-	GroupVersionKind metav1.GroupVersionKind `json:"gvk"`
+	// Additional Information when the condition is bound to a ModuleTemplate. It contains information about the last
+	// parsing that occurred and will track the state of the parser ModuleTemplate in Context of the Installation.
+	// This will update when Channel, Profile or the ModuleTemplate used in the Condition is changed.
```

Comment:
```suggestion
	// This will update when Channel, Profile or the ModuleTemplate used in the Condition is changed.
```
What is this? We should probably change this sentence

Reply:
will update in next commit